node/resolvers/config.ts

```diff
@@ -44,7 +47,7 @@ const checkConfig = async (ctx: Context) => {
       message: 'checkConfig-updatingSchema',
     })
 
-    schemas.forEach((schema) => {
+    schemas.forEach(schema => {
```

Comment:
these changes removing `()` are causing lint erros. Any specific reason to remove it? Otherwise I would add them back to solve the lint errors.